//
// PROJECT:         Aspia
// FILE:            system_info/category_user.proto
// LICENSE:         Mozilla Public License Version 2.0
// PROGRAMMERS:     Dmitry Chapyshev (dmitry@aspia.ru)
//

syntax = "proto3";

option optimize_for = LITE_RUNTIME;

package aspia.proto;

message User
{
    message Item
    {
        string name                  = 1;
        string full_name             = 2;
        string comment               = 3;
        bool is_disabled             = 4;
        bool is_password_cant_change = 5;
        bool is_password_expired     = 6;
        bool is_dont_expire_password = 7;
        bool is_lockout              = 8;
        uint32 number_logons         = 9;
        uint32 bad_password_count    = 10;
        int64 last_logon_time        = 11;
    }

    repeated Item item = 1;
}
